kntest/pkg/cluster/ops/create.go

```diff
@@ -40,6 +39,35 @@ const (
 	projectKey        = "E2E:Project"
 )
 
+// Create creates a GKE cluster and configures gcloud after successful GKE create request
+func Create(o *RequestWrapper) (*clm.GKECluster, error) {
```

Comment:
These should be methods on (o *RequestWrapper)

Reply:
Good point. Done.